hatchways/team-bluejay
# Fe shopping cart

shopping cart loads from local storage, and updated accordingly.

To test,
- add items to shopping cart, cart icon should increase in number.
- add items from a different chef, you should be prompted to empty cart
- refresh page, cart info should still persist from localstorage
- click the shopping cart icon, it should take you to /checkout
- in checkout page, all items should add up correctly with price and quantity
- items can be added or removed from checkout page

To do next:
- make call to checkout w Stripe
